django_setup_configuration/management/commands/check_config_docs.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+
+from ...exceptions import DocumentationCheckFailed
+from .generate_config_docs import ConfigDocBaseCommand
+
+
+class Command(ConfigDocBaseCommand):
```

Comment:
I'm not opposed to this being a separate command, but my initial thought was to implement this as a simple script in CI that runs the generate docs command and then checks if there is a diff after doing that

@annashamray thoughts?

Reply:
I don't mind any approach, but yeah the main purpose is to use it in CI